packages/web-react/src/components/Link/README.md

```diff
@@ -3,19 +3,60 @@
+```
+
+## Underlined
+
+You can customize the underline behavior for links with three different settings:
+
+### Hover
+
+This is the **default** value, which makes the underline visible only when the component is hovered over.
+
+```jsx
+<Link href="/" underlined="hover">
+  …
+</Link>
+```
+
+Alternatively, you can omit this prop:
+
+```jsx
+<Link href="/">…</Link>
+```
+
+### Always
+
+The underline is constantly visible, regardless of interaction.
+
+```jsx
+<Link href="/" underlined="always">
+  …
+</Link>
+```
+
+### Never
+
+The underline is never visible, even when the link is hovered over.
+
+```jsx
+<Link href="/" underlined="never">
+  …
+</Link>
 ```
 
 ## API
 
-| Name           | Type                                             | Default   | Required | Description                    |
-| -------------- | ------------------------------------------------ | --------- | -------- | ------------------------------ |
-| `color`        | [Action Link Color dictionary][dictionary-color] | `primary` | ✕        | Color of the link              |
-| `elementType`  | `ElementType`                                    | `a`       | ✕        | Type of element used as        |
-| `href`         | `string`                                         | —         | ✕        | Link's href attribute          |
-| `isDisabled`   | `bool`                                           | `false`   | ✕        | Whether is the link disabled   |
-| `isUnderlined` | `bool`                                           | `false`   | ✕        | Whether is the link underlined |
-| `ref`          | `ForwardedRef<HTMLAnchorElement>`                | —         | ✕        | Link element reference         |
+| Name           | Type                                             | Default   | Required | Description                                                                          |
+| -------------- | ------------------------------------------------ | --------- | -------- | ------------------------------------------------------------------------------------ |
+| `color`        | [Action Link Color dictionary][dictionary-color] | `primary` | ✕        | Color of the link                                                                    |
+| `elementType`  | `ElementType`                                    | `a`       | ✕        | Type of element used as                                                              |
+| `href`         | `string`                                         | —         | ✕        | Link's href attribute                                                                |
+| `isDisabled`   | `bool`                                           | `false`   | ✕        | Whether is the link disabled                                                         |
+| `isUnderlined` | `bool`                                           | `false`   | ✕        | [**DEPRECATED**][deprecated] in favor of `underline`; Whether is the link underlined |
```

Comment:
please add a separate section with a Deprecation Notice. Like in Twig. 

And here you can link the codemod. :)